docs/content/en/docs/third-party/_index.md

```diff
@@ -8,7 +8,9 @@ description: Third-party software licenses.
 
 K8ssandra provides a production-ready platform for running Apache Cassandra&reg; on Kubernetes. This includes automation for operational tasks such as repairs, backup and restores, and monitoring.
 
-Direct dependencies:
+The K8ssandra software is licensed under the open-source **Apache-2.0 license agreement**. See https://github.com/k8ssandra/k8ssandra/blob/main/LICENSE.
```

Comment:
```suggestion
The K8ssandra software is licensed under the open-source [**Apache-2.0 license agreement**](https://github.com/k8ssandra/k8ssandra/blob/main/LICENSE).
```

Reply:
What do you think about just making the license name a link to the file in the repo?

Reply:
Good idea - will do now...